cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -101,6 +102,34 @@ function(remove_sdk_unsupported_archs name os sdk_path architectures_var)
   set("${architectures_var}" ${architectures} PARENT_SCOPE)
 endfunction()
 
+# Work out which threading package to use by consulting SWIFT_THREADING_PACKAGE
+function(find_threading_package sdk default package_var)
```

Comment:
`get_threading_package` seems like a better name as it is meant to retrieve the value.